src/components/Stats.tsx

```diff
@@ -46,6 +46,11 @@ export const Stats = (props: {
   const selectedElements = getTargetElements(props.elements, props.appState);
   const selectedBoundingBox = getCommonBounds(selectedElements);
 
+  const onGridSizeChange = () => {
+    //TODO: Update the state properly and rerender the grid
+    props.appState.gridSize = ((props.appState.gridSize - 5) % 50) + 10;
```

Comment:
@dwelle What would be the best way to update the state and re-render the canvas for this test?

Reply:
Pass `setAppState` to the component same as we do for `LayerUI`:

https://github.com/excalidraw/excalidraw/blob/86c036505b91bc9d28b869822afe7d507545d7c1/src/components/LayerUI.tsx#L59